modifications/npntcontrollerclass.h

```diff
@@ -0,0 +1,42 @@
+#ifndef NPNTCONTROLLERCLASS_H
+#define NPNTCONTROLLERCLASS_H
+
+#include <QObject>
+#include "QGCApplication.h"
+#include "QTimer"
+#include "QThread"
+#include "keyrotationcontroller.h"
+
+class NpntControllerClass : public QObject
+{
+    Q_OBJECT
+public:
+    explicit NpntControllerClass(QObject *parent = nullptr);
+
+signals:
+      void check1();
```

Comment:
what are the 4 checks? Can we use better names?

Reply:
Basically the checks means, the four checkboxes of the init process as shown below
![image](https://user-images.githubusercontent.com/43859232/137575695-0daaf9bb-79a2-4d8e-b818-ca1d27df5d83.png)
Note a check is hidden for now i.e check for firmware upgrades, that i will implement down the line.

Although, noted I will use better names.

Reply:
solved in this commit 0a01d28